staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1678,6 +1678,17 @@ type ServiceAccountTokenProjection struct {
 	Path string `json:"path" protobuf:"bytes,3,opt,name=path"`
 }
 
+// ClusterTrustBundlePEMProjection allows a pod to access the
+// `.spec.pemTrustAnchors` field of a ClusterTrustBundle object in an
+// auto-updating file.
+type ClusterTrustBundlePEMProjection struct {
```

Comment:
Can we just call this ClusterTrustBundleProjection? We don't say TokenProjectionJWT.

Reply:
We might in the future want to add different formats.  The only one I can actually think of is Java keystores.  Everything else  typically speaks PEM.  We could just give a hypothetical future Java keystore volume a more specific name.

Reply:
Done.